src/assets/WhatWeDo/WhiteWaveBlock.css

```diff
@@ -0,0 +1,193 @@
+}
+
+.whitewaveblock .whitewaveblock-wave {
+    width: 100%;
+    height: 100%;
+}
+
+.whitewaveblock {
+    position: relative;
+    display: flex;
+    margin-top: 0px;
+    z-index: -1;
+    background-color: var(--white);
+    padding-left: 20%;
+    padding-right: 20%;
+    padding-top: 4.5rem;
+}
+
+.whitewaveblock img {
+    width: 80%;
+    max-width: 520px;
+    min-width: 450px;
+}
+
+.whitewaveblock-img {
+    padding-left: 0.2rem;
+    width: 60%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.whitewaveblock-cta {
+    width: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.whitewaveblock-wave {
+    position: relative;
+    margin-top: -10em;
+    z-index: -10;
+}
+
+.break {
+    display: none;
+}
+
+@media screen and (max-width: 1700px) {
+
+    .whitewaveblock {
+        padding-left: 18%;
+        padding-right: 18%;
+        
+    }
+
+    .whitewaveblock-cta {
+        transform: scale(0.9);
+    }
+
+    .whitewaveblock-wave {
+        position: relative;
+        margin-top: -6em;
+        z-index: -10;
+    }
+
+}
+
+@media screen and (max-width: 1450px) {
+
+    .whitewaveblock {
+        padding-left: 10%;
+        padding-right: 10%;
+        display: flex;
+        flex-direction: column-reverse;
+    }
+
+}
+
+@media screen and (max-width: 1300px) {
+
+    .whitewaveblock {
+        padding-left: 8%;
+        padding-right: 8%;
+        display: flex;
+        flex-direction: column-reverse;
+    }
+
+    .whitewaveblock-cta {
+        transform: scale(0.8);
+    }
+
+    .whitewaveblock-img {
+        padding-left: 1rem;
+    }
+
+    .whitewaveblock img {
+        width: 80%;
+        min-width: 400px;
+    }
+
+}
+
+@media screen and (max-width: 1100px) {
+/*
+    .whitewaveblock-wave {
+        margin-top: -10em;
+    }
+*/
+
+    .whitewaveblock {
+        padding-top: 2.5rem;
+        display: flex;
+        flex-direction: column-reverse;
+    }
+
+}
+
+/* 860 */
+
+@media screen and (max-width: 860px) {
+
+    .whitewaveblock {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: center;
+        margin-left: 0;
+        margin-right: 0;
+        display: flex;
+        flex-direction: column-reverse;
+        /*
+        position: relative;
+        display: flex;
+        margin-top: 0px;
+        z-index: -1;
+        background-color: #CAF0F8;
+        padding: 20px;
+        */
+    }
+
+    .break {
+        display: block;
+        flex-basis: 100%;
+        height: 0;
+    }
+
+    .whitewaveblock-cta {
+        width: 100%;
+        padding: 0;
+    }
+
+    .whitewaveblock-img {
+        padding-left: 0;
+    }
+
+    .whitewaveblock-img img{
+        width: 80%;
+        max-width: 250px;
+        min-width: 160px;
+    }
+
+}
+
+@media screen and (max-width: 750px) {
+
+    .whitewaveblock {
+        height: 490px;
+        display: flex;
+        flex-direction: column-reverse;
```

Comment:
You only need to specify a style once — for example, since you specified 
`
display: flex;
flex-direction: column-reverse;
' 
for `.whitewaveblock' in this media query, you don't need to do it in the media query below bc 640 is included in 750.

Reply:
I see, thanks for the information!